src/auth.js

```diff
@@ -217,28 +359,25 @@ class Auth {
    */
   createUserProfile(fields, result) {
     const ignores = ['password'];
-    const required = ['username', 'id'];
```

Comment:
Why was `id` removed here ?

Reply:
I honestly don't remember, I can add it back if you would prefer. I think I got rid of all the `id` code in my fork since it was seemingly unused, I just use the username as the unique identifier.

Reply:
Why `id` was put there in the first place was because I wanted to have a way to query things throughout the stack. And all auth adapters should return an id if possible (or just fall back to using the username there or something).

Just having a "username" sometimes is not enough to do authentication or authorization with external systems.

Reply:
Sounds good, I can add that back no problem. Thanks for the context.

Reply:
Just had to check if that was in the manual, and it seems like I remembered to put that in :sweat_smile: 

> Please note that the OS.js client expects to receive an JSON object with at least {id, username}.